### PR TITLE
Vue component refactor

### PR DESCRIPTION
**Pull request by BoDonkey**
## Summary

*Summarize the changes briefly, including which issue/ticket this resolves. If it closes an existing Github issue, include "Closes #[issue number]"*
This PR attempts to optimize the logic of the Character Control module components by moving some of the duplicated logic to a mixin. Additionally, the package name was corrected in the `package.json` file to match the new repo name. This code should be looked at for functionality and with an eye toward being used in a tutorial. So, if it  maintains function, but is not a good code practice, it should be flagged (or outright rejected).

## What are the specific steps to test this change?

*For example:*
> 1. Run the website and log in as an admin
> 2. Open a piece manager modal and select several pieces
> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
> 4. Check that all pieces have been archived properly

1. Install the modules in an A3 project.
2. Add the `characterCount` tool and insert to a rich-text-widget-configuration. 
3. Check for functionality by the toolbar button or insert menu item.
## What kind of change does this PR introduce?
*(Check at least one)*

- [ ] Bug fix
- [ ] New feature
- [X] Refactor
- [ ] Documentation
- [ ] Build-related changes
- [ ] Other

## Make sure the PR fulfills these requirements:

- [X] It includes a) the existing issue ID being resolved, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves
- [ ] The changelog is updated
- [ ] Related documentation has been updated
- [ ] Related tests have been updated

If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: modules/@apostrophecms/characterCount/index.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ module.exports = {
         const finalTools = {
           ...initialData.tools,
           characterCount: {
-            component: 'TiptapCharacterCount',
+            component: 'AposCCToolbar',
```
Comment (boutell) [inline modules/@apostrophecms/characterCount/index.js:14]: If this isn't going to ship as a feature per se, then we shouldn't use the Apos prefix as that could cause the example to collide with something we release later.

File: modules/@apostrophecms/characterCount/ui/apos/components/AposCCInsert.vue [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+@import '../scss/characterCountStyles';
+
+.apos-context-menu__dialog {
+    width: 500px;
+  }
+
+.apos-cc-insert__dialog {
+  z-index: $z-index-modal;
+  position: absolute;
+  top: calc(100% + 5px);
```
Comment (boutell) [inline modules/@apostrophecms/characterCount/ui/apos/components/AposCCInsert.vue]: Huh. Where did this math come from?

**Review by boutell — CHANGES_REQUESTED** [review]

File: modules/@apostrophecms/characterCount/ui/apos/components/AposCCInsert.vue (hunk 2 above)
Comment (BoDonkey) [reply modules/@apostrophecms/characterCount/ui/apos/components/AposCCInsert.vue]: https://github.com/apostrophecms/ai-helper/blob/67a8562f591b28ed5e81ecc064e739c2467e7d55/modules/%40apostrophecms/ai-helper-rich-text-widget/ui/apos/components/AposAiHelperTextDialog.vue#L120

File: modules/@apostrophecms/characterCount/index.js (hunk 1 above)
Comment (BoDonkey) [reply modules/@apostrophecms/characterCount/index.js:14]: Changed.

**Review by boutell — APPROVED** [review]

**Pull request merged**